examples/clone/clone.bal

```diff
@@ -36,34 +35,31 @@ public function main() returns (Person, Person, string)? {
         address: address
     };
 
-    // Assign the `person` value to `anyValue`. So the type of the value has to be determined in runtime.
+    // Assign the `person` value to an `any` typed variable `anyValue`.
+    // The type of the value held by `anyValue` will be determined at run time.
     any anyValue = person;
 
-    // Invoke the clone builtin method. The return type is either `Person` or `error`. Note that the `error` is returned
-    // if the value being cloned is not of `anydata` type. If the type of the value being cloned can be determined in
-    // compile time, return value is exactly of the type of the value being cloned.
-    // e.g. Person result = person.clone();
-    // In above example, 'person' is cloned instead of 'anyValue'. In compile time itself, we know the return type
-    // should be 'Person'
+    // Invoke the `.clone()` built-in method. The return type is either `Person` or `error`. Note that `error` could be 
+    // returned if the variable on which `.clone()` is called is not of type `anydata`.
+    // If the type of the value being cloned can be determined at compile time and is `anydata`, the type of the
```

Comment:
Out of my curiosity :) , shouldn't it be "If the type of the value is being" ?

Reply:
Do you mean 
```suggestion
    // If the type of the value that is being cloned can be determined at compile time and is `anydata`, the type of the
```

Reply:
Let's change it to: "If the type of the value that is cloned..."

That removes any discrepancies with the tense used here.

Reply:
```suggestion
    // If the type of the value that is cloned can be determined at compile time and is `anydata`, the type of the
```

Reply:
The way this is worded is strange because of the compile time bit. I think the fact that this happens during compile time shouldn't be part of the if clause.

Shall we change it to "At compile time, if the type of the value that is cloned is `anydata`, the type of the return value is exactly the type of the value being cloned."